src/ngAutocomplete.js

```diff
@@ -148,6 +148,12 @@ angular.module( "ngAutocomplete", [])
           }
         }
 
+        if (scope.options.getPlaceOnBlur) {
```

Comment:
Should check options presence.
`if ( scope.options && scope.options.getPlaceOnBlur) {`